shared/sh_shared.lua

```diff
@@ -19,12 +27,20 @@ Shared.WeedProps = {
     [5] = `bkr_prop_weed_lrg_01b`
 }
 
+Shared.ProcessingProps = {
+    {model = "bkr_prop_weed_table_01a",    coords = vector4(1045.41, -3197.64, -38.13, 270),},
+    {model = "hei_prop_heist_weed_pallet", coords = vector4(1044.89, -3192.6, -37.91, 196.56),},
+    {model = "hei_prop_heist_weed_pallet", coords = vector4(1043.07, -3192.55, -37.91, 183.94),},
+    {model = "hei_prop_heist_weed_pallet", coords = vector4(1041.27, -3192.61, -37.91, 183.98),},
+    {model = "hei_prop_heist_weed_pallet", coords = vector4(1042.64, -3206.14, -37.85, 21.93),},
+}
+
 Shared.PackageProp = `prop_mp_drug_package`
 
 --- Growing Related Settings
 Shared.rayCastingDistance = 7.0 -- distance in meters
 Shared.ClearOnStartup = true -- Clear dead plants on script start-up
-Shared.ObjectZOffset = - 0.5 -- Z-coord offset for WeedProps
+Shared.ObjectZOffset = - 0.01 -- Z-coord offset for WeedProps
```

Comment:
I would keep this offset a bit higher than 0.01, maybe 2 or 3

Reply:
When I was using the default value the plant was planting underground, so I made this 0.01 which perfectly sets to the ground.

But what do you want me to do 0.2 or 0.02?

Reply:
> When I was using the default value the plant was planting underground, so I made this 0.01 which perfectly sets to the ground.
> 
> But never mind I set it to 0.2..

Yeah but that's because the plant is supposed to go a little above ground, it's on a container not directly planted in the ground, not sure if that makes sense.